bitnami/cassandra/templates/statefulset.yaml

```diff
@@ -290,12 +321,25 @@ spec:
         - name: metrics
           image: {{ include "cassandra.metrics.image" . }}
           imagePullPolicy: {{ .Values.metrics.image.pullPolicy | quote }}
+          {{- if .Values.hostNetwork }}
+          env:
+            - name: CASSANDRA_EXPORTER_CONFIG_listenPort
+              value: {{ .Values.metrics.containerPorts.http | quote }}
+            - name:  CASSANDRA_EXPORTER_CONFIG_host
+              value: localhost:{{ .Values.containerPorts.cassandra.jmx }}
```

Comment:
Just curious: why this weird "FIRST UPPERCASE, then lowercase" syntax for the environment variables?

Reply:
Yeah .-)
This in "preperation" for the `cassandra-exporter` because bitnami references the https://github.com/criteo/cassandra_exporter in https://github.com/bitnami/bitnami-docker-cassandra-exporter

An criteo gives actual the ability to set the config over this "UPPER lower" Case-Thing ->
https://github.com/criteo/cassandra_exporter/blob/master/docker/run.sh

Maybe a question from my side: Can we also start the bitnami/metric Container with a "launch" Script which is able to edit exporter.config? 

TL;DR
Both variables CASSANDRA_EXPORTER_CONFIG_host and CASSANDRA_EXPORTER_CONFIG_listenPort are only necessary for HOST-Network=true and set also the critord/cassandra-exporter image instead of the bitname/cassandra-exporter

Reply:
> An criteo gives actual the ability to set the config over this "UPPER lower" Case-Thing ->
https://github.com/criteo/cassandra_exporter/blob/master/docker/run.sh

That's fair enough! It's "weird" they decided to use that syntax but it's not a major issue.

> Maybe a question from my side: Can we also start the bitnami/metric Container with a "launch" Script which is able to edit exporter.config?

I don't know if it's worth to do that for a simple container such an exporter.